packages/wonder-stuff-testing/src/jest/isolate-modules.js

```diff
@@ -13,10 +13,10 @@ import {assertJest} from "./internal/assert-jest.js";
  * discourage dynamic `import` use, which doesn't play well with standard
  * jest yet.
  */
-export const isolateModules = <T>(action: () => T): T => {
+export const isolateModules = <T>(action: () => ?T): ?T => {
```

Comment:
Huh, why did this change? Wouldn't this also require a minor version update for the wonder-stuff-testing package?

Reply:
I am honestly not sure. Flow started complaining about it - I can only assume some auto-update of flow missed it before.

The code already worked this way so it's only a type change; existing code shouldn't behave any differently from this so I didn't feel a minor update was worth it.